api/src/functionalTest/java/uk/gov/hmcts/payment/functional/RefundsRequestorJourneyPBAFunctionalTest.java

```diff
@@ -569,10 +570,12 @@ public void positive_add_remission_and_add_refund_for_2_pba_payments() {
             .then().getResponse();
         String remissionReference = response.getBody().jsonPath().getString("remission_reference");
 
+        RetrospectiveRemissionRequest retrospectiveRemissionRequest
+            = PaymentFixture.aRetroRemissionRequest(remissionReference);
+
         Response refundResponse = paymentTestService.postSubmitRefund(USER_TOKEN_PAYMENTS_REFUND_REQUESTOR_ROLE,
-            SERVICE_TOKEN_PAYMENT,
-            RetrospectiveRemissionRequest.retrospectiveRemissionRequestWith().remissionReference(remissionReference).contactDetails(
-                    ContactDetails.contactDetailsWith().build()).build());
+            SERVICE_TOKEN_PAYMENT, retrospectiveRemissionRequest);
+
         assertThat(refundResponse.statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

Comment:
We need to add a negative test that checks on the Error scenarios of the Contact Details.make sure we are getting the right HTTPS Response Codes and reasons as well.

Reply:
A bare miniumum of the Contact Details as per Acceptance Criteria should be given...